ledger/participant-integration-api/src/main/scala/platform/index/BuffersUpdater.scala

```diff
@@ -57,9 +62,15 @@ private[index] class BuffersUpdater(
           randomFactor = 0.0,
         )
       )(() => subscribeToTransactionLogUpdates(updaterIndex.get))
-      .map { case ((offset, eventSequentialId), update) =>
-        updateCaches(offset, update)
-        updaterIndex.set(Some(offset -> eventSequentialId))
+      .async
+      .mapAsync(1) { case ((offset, eventSequentialId), update) =>
+        Timed.future(
+          metrics.daml.index.updateCaches,
```

Comment:
This metric is used asymmetrically. Here it covers both the transactions buffer as well as mutable state cache. Further down it covers only the latter. I suggest either splitting by domain i.e. msc vs tb or by location i.e. BuffersUpdater vs MutableCacheBackedContractStore

Reply:
The update paths are used disjointly and having both under the same name might actually benefit metrics visualization (i.e. you don't need to register two different metrics based on which feature is enabled). wdyt?

Reply:
Ok scratch that. Changed my mind